libs/blocks/nav-marquee/nav-marquee.css

```diff
@@ -0,0 +1,278 @@
+.nav-marquee {
+  flex: 1 1 auto;
+}
+
+.nav-marquee.light {
+  color: var(--text-color);
+  background-color: #FFF;
+}
+
+.nav-marquee.dark {
+ background-color: #000;
```

Comment:
To fully support `.dark` we need to consider the section so
```
.dark .nav-marquee, 
.nav-marquee.dark
```
for when it's nested in a .section.dark
Also shouldn't the font color be included here?